src/components/ZclAttributeManager.vue

```diff
@@ -276,6 +276,17 @@ export default {
         return false
       }
     },
```

Comment:
When do you use isDisabled v/s isDisabledDefault ?

Reply:
The default value should be disabled whenever the Storage is set to "External".

In a case where the user sets Storage to External but wants to change it back to RAM then it should be editable while the default value is uneditable.